src/TorchVision/File.cs

```diff
@@ -58,6 +59,7 @@ public static void write_file(string filename, Tensor data)
             /// <param name="data">One dimensional <c>uint8</c> <cref>Tensor</cref>.</param>
             public static async void write_file_async(string filename, Tensor data)
             {
+                // Currently limited to 2GB - should we duplicate the code for WriteBytesToStream using async ops?
```

Comment:
Comment should be deleted before merge - wrote it for @NiklasGustafsson

Reply:
OK

Reply:
Should we write an async function?